mobile/header.html

```diff
@@ -61,42 +61,43 @@
     }
   });
 
-  api.reopenWidget('brand-header', {
-  
-    defaultState() {
-      let states =  {
-        brandMenuVisible: false
-      };
-  
-      return states;
-    },
-  
-    toggleBrandMenu() {
-      this.state.brandMenuVisible = !this.state.brandMenuVisible;
+  if (!settings.show_bar_on_mobile) {
+    api.reopenWidget('brand-header', {
+      defaultState() {
+        let states =  {
+          brandMenuVisible: false
+        };
 
-      if (!this.state.brandMenuVisible) {
-        const $headerCloak = $(".header-cloak");
-        $headerCloak.removeClass("animate");
-        $headerCloak.hide();
-        $headerCloak.attr("style", "");
-      }
-    },
-  
-    html(attrs, state) {
-      const { siteSettings } = this;
-      const panelContents = [];
-  
-      panelContents.push(this.attach('brand-header-icon', { brandMenuVisible: state.brandMenuVisible }));
-
-      if (state.brandMenuVisible) {
-        panelContents.push(this.attach('hamburger-brand-menu', { generalLinks: this.generalLinks(), iconLinks: this.iconLinks() }));
+        return states;
+      },
+
+      toggleBrandMenu() {
+        this.state.brandMenuVisible = !this.state.brandMenuVisible;
+
+        if (!this.state.brandMenuVisible) {
+          const $headerCloak = $(".header-cloak");
```

Comment:
While you're in here it would be nice to remove jQuery. Not important too pass review, though.